tests/ImageSharp.Tests/TestUtilities/Attributes/Intrinsics/IntrinsicTestsUtils.cs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using System.ComponentModel;
+#if SUPPORTS_RUNTIME_INTRINSICS
+using System.Runtime.Intrinsics.X86;
+#endif
+
+namespace SixLabors.ImageSharp.Tests
+{
+    public static class IntrinsicTestsUtils
+    {
+        public static _HwIntrinsics GetNotSupportedIntrinsics(this _HwIntrinsics flags)
+        {
+            _HwIntrinsics notSupported = flags;
+
+#if SUPPORTS_RUNTIME_INTRINSICS
+            // SSE
+            UncheckIfSupported(ref notSupported, _HwIntrinsics.SSE, Sse.IsSupported);
+            UncheckIfSupported(ref notSupported, _HwIntrinsics.SSE2, Sse2.IsSupported);
+            UncheckIfSupported(ref notSupported, _HwIntrinsics.SSE3, Sse3.IsSupported);
+            UncheckIfSupported(ref notSupported, _HwIntrinsics.SSSE3, Ssse3.IsSupported);
+            UncheckIfSupported(ref notSupported, _HwIntrinsics.SSE41, Sse41.IsSupported);
+            UncheckIfSupported(ref notSupported, _HwIntrinsics.SSE42, Sse42.IsSupported);
+
+            // AVX
+            UncheckIfSupported(ref notSupported, _HwIntrinsics.AVX, Avx.IsSupported);
+            UncheckIfSupported(ref notSupported, _HwIntrinsics.AVX2, Avx2.IsSupported);
+
+            // BMI
+            UncheckIfSupported(ref notSupported, _HwIntrinsics.BMI1, Bmi1.IsSupported);
+            UncheckIfSupported(ref notSupported, _HwIntrinsics.BMI2, Bmi2.IsSupported);
+
+            // ABM
+            UncheckIfSupported(ref notSupported, _HwIntrinsics.POPCNT, Popcnt.IsSupported);
+            UncheckIfSupported(ref notSupported, _HwIntrinsics.LZCNT, Lzcnt.IsSupported);
+
+            // FMA
+            UncheckIfSupported(ref notSupported, _HwIntrinsics.FMA, Fma.IsSupported);
+
+            // AES
+            UncheckIfSupported(ref notSupported, _HwIntrinsics.AES, Aes.IsSupported);
+
+            // PCLMULQDQ
+            UncheckIfSupported(ref notSupported, _HwIntrinsics.PCLMULQDQ, Pclmulqdq.IsSupported);
+#endif
+
+            return notSupported;
+        }
+
+#if SUPPORTS_RUNTIME_INTRINSICS
+        private static void UncheckIfSupported(ref _HwIntrinsics set, _HwIntrinsics value, bool isSupported)
+        {
+            if (isSupported)
+            {
+                set &= ~value;
+            }
+        }
+#endif
+    }
+
+    [Flags]
+    public enum _HwIntrinsics
```

Comment:
This goes against our naming conventions, which are for some reason not strictly checked now (see #1658).

I think this could be more general than just "HW intrinsics", maybe let's name it `RuntimeFeatureFilter` ?

Reply:
Yeah, it's checked by stylecop, I named it this way because `FeatureTestRunner` already has enum with this name but it's used for disabling features. `RuntimeFeatureFilter` looks cleaner considering Vector8 stuff, good suggestion! Will fix asap.